Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -39,20 +39,28 @@ enum ConfigCLI {
   }
 
   // MARK: Run
-
+  
   static let run = command(
     CLIOption.configFile(),
+    Flag("quiet", default: false, flag: "q", description: "Hide all non error logs"),
     Flag("verbose", default: false, flag: "v", description: "Print each command being executed")
-  ) { file, verbose in
+  ) { file, quiet, verbose in
     do {
       try ErrorPrettifier.execute {
         let config = try Config(file: file)
-
-        if verbose {
-          logMessage(.info, "Executing configuration file \(file)")
+        
+        switch (quiet, verbose) {
+        case (true, _):
+            commandLogLevel = .quiet
+        case (_, true):
+            commandLogLevel = .verbose
+            logMessage(.info, "Executing configuration file \(file)")
```

Comment:
Move this out (after) of the switch, it won't be displayed when the level is set accordingly.

Reply:
Oops. Done.